airbyte-integrations/connectors/source-freshsales/acceptance-test-config.yml

```diff
@@ -20,6 +20,13 @@ acceptance_tests:
       - config_path: "secrets/config.json"
         expect_records:
           path: "integration_tests/expected_records.jsonl"
+        empty_streams:
+          - name: "sales_accounts_filter"
```

Comment:
are the tests failing on master too?

Reply:
The master branch does not have these filter streams exposed to the user. The filter streams are used only internally in the manifest.
However when using the builder, if I want to use the filter streams as a parent stream for another stream, I need to expose the streams. So basically the user will be able to see 3 new streams. But we don't want the user to use these streams. 
Having a checkbox such as `use internally only` in the builder would be a nice addition where the stream will not be exposed to the user, but we could still be able to use the stream as a parent stream for another stream.